silverstripe/comments/SS-2018-015-1.yaml

```
@@ -4,10 +4,10 @@ cve:       ~
 branches:
     1.x:
         time:     null
-        versions: ['>=1.3.0', '<1.99.99']
+        versions: ['>=1.3.0', '<1.9.9']
     2.x:
         time:     null
-        versions: ['>=2.0.0', '<2.99.99']
+        versions: ['>=2.0.0', '<2.9.9']
```

```suggestion
        versions: ['>=2.0.0', '<2.9.99']
```

Actually, it looks like other contributors are following this format. I'd feel safer about this TBH.